src/pyhf/cli/contrib.py

```diff
@@ -0,0 +1,71 @@
+"""CLI for functionality that will get migrated out eventually."""
+import logging
+import click
+from pathlib import Path
+
+from ..contrib import utils
+
+logging.basicConfig()
+log = logging.getLogger(__name__)
+
+
+@click.group(name="contrib")
+def cli():
+    """
+    Contrib experimental operations.
+
+    .. note::
+
+        Requires installation of the ``contrib`` extra.
+
+        .. code-block:: shell
+
+            $ python -m pip install pyhf[contrib]
+    """
+
+
+@cli.command()
+@click.argument("archive-url", default="-")
+@click.argument("output-directory", default="-")
+@click.option("-v", "--verbose", is_flag=True, help="Enables verbose mode")
+@click.option(
+    "-f", "--force", is_flag=True, help="Force download from non-approved host"
+)
+@click.option(
+    "-c",
+    "--compress",
+    is_flag=True,
+    help="Keep the archive in a compressed tar.gz form",
+)
+def download(archive_url, output_directory, verbose, force, compress):
+    """
+    Download the patchset archive from the remote URL and extract it in a
+    directory at the path given.
+
+    Example:
+
+    .. code-block:: shell
+
+        $ pyhf contrib download --verbose https://www.hepdata.net/record/resource/1408476?view=true 1Lbb-likelihoods
```

Comment:
should we make the url be the `doi.org` one?

Reply:
@lukasheinrich I would love that, but we don't have a DOI for that likelihood tarball. Do you know of one?